docs/user-guide/deploying-application/triggering-ci.md

```diff
@@ -1,41 +1,34 @@
-# Triggering CI
+# Triggering CI Pipelines
 
-## Triggering CI Pipelines
+To trigger the CI pipeline, first you need to select the Git commit for which the CI pipeline will be triggered. To select the Git commit, click on the `Select Material` button present on the CI pipeline.
 
-The CI Pipeline can be triggered by selecting `Select Material`
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-ci/select-material-new.jpg)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-ci/select-material.jpg)
+Once clicked, a list will appear showing various commits made in the repository, including details such as the author name, commit date and time etc. Choose the desired commit for which you want to trigger the pipeline, and then click on "Start Build" to initiate the CI pipeline.
 
-CI Pipelines that are set as automatic are always triggered as soon as a new commit is made to the git branch they're sensing. However, CI pipelines can always be manually triggered as and if required.
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-ci/trigger-build.jpg)
 
-Various commits done in the repository can be seen, here along with details like Author, Date etc. Select the commit that you want to trigger and then click on `Start Build` to trigger the CI pipeline.
+CI Pipelines with automatic triggers are triggereded immediately when a new commit is made to the git branch. If the trigger for a build pipeline is set to manual, it will not be automatically triggered and requires manual trigger.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-ci/start-build-1.jpg)
-
-
-**Refresh** icon, refreshes Git Commits in the CI Pipeline and fetches the latest commits from the “Repository”
+The **Refresh** icon updates the Git Commits section in the CI Pipeline by fetching the latest commits from the repository. Clicking on the refresh icon ensures that you have the most recent commit available.
 
 **Ignore Cache** : This option will ignore the previous build cache and create a fresh build. If selected, will take a longer build time than usual.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploying-application/triggering-ci/build-history.jpg)
-
-It can be seen that the pipeline is triggered here and is the _Running_ state.
-
-Click on your `CI Pipeline` or click on `Build History` to get the details about the CI pipeline such as logs, reports etc.
+Click on the `CI Pipeline` or navigate to the `Build History` to get the CI pipeline details such as build logs, sorce code details, artifacts and vulnerability scan reports.
```

Comment:
Click the `CI Pipeline` or navigate to the `Build History` to get the CI pipeline details such as build logs, source code details, artifacts, and vulnerability scan reports.